tests/test_pond.py

```diff
@@ -60,18 +228,30 @@ def test_pond_basic(data_width=16,  # CGRA Params
 
     data_in_pond = [0] * interconnect_input_ports
     valid_in = [0] * interconnect_input_ports
-    for i in range(32):
+    for i in range(24):
         # Incrementing Data
         data_in_pond[0] = data_in_pond[0] + 1
 
         if interconnect_input_ports == 1:
-            tester.circuit.data_in_pond = data_in_pond[0]
+            if dsl:
```

Comment:
can we opt for passing the name in or figuring it out without if/else everywhere for the dsl flag?

Reply:
yeah, I'll do it at the top if dsl is true and just use one name throughout so it's cleaner.

Reply:
done.